llama_index/vector_stores/weaviate.py

```diff
@@ -205,6 +205,7 @@ def delete(self, ref_doc_id: str, **delete_kwargs: Any) -> None:
             self._client.query.get(self.index_name)
             .with_additional(["id"])
             .with_where(where_filter)
+            .with_limit(10000)
```

Comment:
Can we add a note about why `10000` is used here? I will definitely forget in a week lol